packages/chatbot-server-mongodb-public/src/processors/makeMongoDbReferences.ts

```diff
@@ -75,6 +75,7 @@ export function mongodbReferenceType(
       case "practical-aggregations-book":
         return "Book";
       case "devcenter":
+        return "Article";
```

Comment:
relevant tests now failing b/c of this change. pls update test to match

Reply:
Updated